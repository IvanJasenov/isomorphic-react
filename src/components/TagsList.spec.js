import React from 'react';
<<<<<<< HEAD
import TagsList from './TagsList'
import renderer from 'react-test-renderer';


describe("The tags list",()=>{
    /**
     * The tagsList can be tested against an expected snapshot value, as in below.
     */
   it ("renders as expected",()=>{
       const tree = renderer
           .create(<TagsList tags={[`css`,`html`,`typescript`,`coffeescript`]}/>)
           .toJSON();

       expect(tree).toMatchSnapshot();
   });
=======
import TagsList from './TagsList';
import renderer from 'react-test-renderer';

describe("The tags list",()=>{
    it("renders as expected",()=>{
        const tree = renderer
            .create(<TagsList tags={[`css`,`html`,`swift`]} />)
            .toJSON();

        console.log(tree);

        expect(tree).toMatchSnapshot();
    });
>>>>>>> a56a6c9a9726fdf2c60b1642c3c5bfe46dcbfeeb
});