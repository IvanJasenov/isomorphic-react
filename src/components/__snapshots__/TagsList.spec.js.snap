// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The tags list renders as expected 1`] = `
<div>
  <code>
    css
  </code>
  <code>
    html
  </code>
  <code>
<<<<<<< HEAD
    typescript
  </code>
  <code>
    coffeescript
=======
    swift
>>>>>>> a56a6c9a9726fdf2c60b1642c3c5bfe46dcbfeeb
  </code>
</div>
`;
